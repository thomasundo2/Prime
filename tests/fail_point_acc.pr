int main()
{
    pt a;
    a = [1l, 2l];
    printl(a.z);
}