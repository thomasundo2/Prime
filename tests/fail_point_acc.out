Fatal error: exception Failure("invalid access element z in a.z")
